### DISCUSSION TITLE
Install firmware Pybricks on my HUB 51515

### DISCUSSION THREAD

**Discussion by jhguay** [pybricks/support#1076] [General]
I see Lego technic Large Hub in DFU Mode in other Devices, but I cant find  update good pilot for my windows 10. Help please.

**Comment by dlech** [comment]
Can you please give the exact steps to reproduce the problem?

**Reply by jhguay** [reply]
OK, I see my error. Sorry. It's work.  Next problem: how to receive a message from my PC.  I use Thonny on my PC.
